src/cli.rs

```diff
@@ -1,3 +1,60 @@
 //! cli
+#![deny(non_upper_case_globals)]
+#![deny(non_camel_case_types)]
+#![deny(non_snake_case)]
+#![deny(unused_mut)]
+#![deny(missing_docs)]
 
-pub(crate) mod node;
+use crate::core::actors;
+use ctrlc;
+use failure;
+
+use std::path::PathBuf;
+use std::result::Result;
+use structopt::StructOpt;
+
+/// Witnet network
+#[derive(Debug, StructOpt)]
+pub(crate) struct Cli {
+    /// `witnet cmd ...`
+    #[structopt(subcommand)]
+    pub(crate) cmd: Command,
+}
+
+#[derive(Debug, StructOpt)]
+pub(crate) enum Command {
+    #[structopt(name = "node", about = "Run the Witnet server")]
+    Node {
+        /// TCP address to which the server should build
+        #[structopt(name = "address", short = "d")]
+        address: String,
+
+        /// Address to peer connection
+        #[structopt(name = "peer", short = "p")]
+        peer: String,
+
+        /// Config file path
+        #[structopt(name = "config", short = "c")]
+        #[structopt(parse(from_os_str))]
+        config: Option<PathBuf>,
+
+        /// Run the server in the background
+        #[structopt(name = "background", short = "b")]
+        background: bool,
+    },
```

Comment:
Currently only the `config` parameter is used, the other ones are not implemented so they can be removed to avoid confusion.